platform/broadcom/saibcm-modules/sdklt/linux/knetcb/psample-cb.c

```diff
@@ -218,6 +218,7 @@ psample_task(struct work_struct *work)
     unsigned long flags;
     struct list_head *list_ptr, *list_next;
     psample_pkt_t *pkt;
+    struct psample_metadata md = {0};
```

Comment:
Remove line 221.  The psample_metadata struct is added below on line 235 which is only enabled with the proper config macros.